src/newTest/java/org/opensearch/node/PluginAwareNode.java

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
Is this license header correct?

Reply:
Yes. Actually, the same file is already located at https://github.com/opensearch-project/security/blob/main/src/test/java/org/opensearch/node/PluginAwareNode.java 

We just needed to duplicate it in order to keep a strict separation between old and new tests.

Reply:
Why do they need to be isolated?

Reply:
When we come to the point that we can just replace the legacy tests by the new tests, it is easier to have a strict separation.